### PR TITLE
docs(react-native): add inferred targets for react native

### PR DESCRIPTION
**Pull request by xiongemi**
## Related Issue(s)

Fixes #

### CODE REVIEW

**Review by isaacplmann — CHANGES_REQUESTED** [review]
Some search and replace typos.  Otherwise looks good

File: docs/generated/packages/expo/documents/overview.md [hunk 1]
```diff
@@ -1,47 +1,116 @@
 Expo is an open-source framework for apps that run natively on Android, iOS, and the web. Expo brings together the best of mobile and the web and enables many important features for building and scaling an app.
 
-Expo is a set of tools built on top of React Native. The Nx Plugin for Expo contains generators for managing Expo applications and libraries within an Nx workspace.
+Expo is a set of tools built on top of Expo. The Nx Plugin for Expo contains generators for managing Expo applications and libraries within an Nx workspace.
```
[the remark this answers is not in the document]
Comment (xiongemi) [reply docs/generated/packages/expo/documents/overview.md:3]: changed

File: docs/shared/packages/expo/expo-plugin.md [hunk 2]
```diff
@@ -1,47 +1,116 @@
 Expo is an open-source framework for apps that run natively on Android, iOS, and the web. Expo brings together the best of mobile and the web and enables many important features for building and scaling an app.
 
-Expo is a set of tools built on top of React Native. The Nx Plugin for Expo contains generators for managing Expo applications and libraries within an Nx workspace.
+Expo is a set of tools built on top of Expo. The Nx Plugin for Expo contains generators for managing Expo applications and libraries within an Nx workspace.
```
[the remark this answers is not in the document]
Comment (xiongemi) [reply docs/shared/packages/expo/expo-plugin.md]: changed

File: docs/shared/packages/expo/expo-plugin.md [hunk 3]
```diff
@@ -238,7 +307,7 @@ nx install <app-name> --fix
 
 ### Run an EAS Build
 
-Expo Application Services (EAS) are deeply integrated cloud services for Expo and React Native apps. EAS Build is a hosted service for building app binaries for your Expo and React Native projects.
+Expo Application Services (EAS) are deeply integrated cloud services for Expo and Expo apps. EAS Build is a hosted service for building app binaries for your Expo and Expo projects.
```
[the remark this answers is not in the document]
Comment (xiongemi) [reply docs/shared/packages/expo/expo-plugin.md]: changed

**Review by isaacplmann — APPROVED** [review]

**Pull request merged**